assert/assertions.go

```diff
@@ -410,9 +411,9 @@ func Equal(t TestingT, expected, actual interface{}, msgAndArgs ...interface{})
 		return Fail(t, fmt.Sprintf("Invalid operation: %#v == %#v (%s)",
 			expected, actual, err), msgAndArgs...)
 	}
-
+	coloredOutput, _ := strconv.Atoi(os.Getenv("TESTIFY_COLORED_DIFF"))
```

Comment:
it might be nice to store this value on the suite once after evaluation so we don't have to check on every `Equal` call.

Additionally, how do you feel about using a function to recognize `yes`|no and `true|false` as well.